extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -505,9 +505,8 @@ export class AmpA4A extends AMP.BaseElement {
    * signing service(s) is to be used.
    * @return {!Array<string>} A list of signing services.
    */
-  getSigningServiceNames() {
-    // TODO(levitzky) Add dev key name once it goes live.
-    return getMode().localDev ? ['google'] : ['google'];
+  getSigningServiceNames_() {
+    throw new Error('getSigningServiceNames not implemented!');
```

Comment:
I'm not sure. My reasoning was that it would be best to remain as network-independent as possible in A4A. But @keithwrightbos can correct me if he thinks otherwise.